utils.py

```diff
@@ -82,8 +81,8 @@ def arg_parser() -> argparse.Namespace:
         "-p",
         "--path",
         help="Define the pathname of flight data to use in mock simulation. Interest Launch data is used by default",
-        type=Path,
-        default="scripts/imu_data/InterestLaunch-9-28.csv",
+        type=str,
+        default="launch_data/interest_launch_9_28.csv",
```

Comment:
why not a Path object?

Reply:
I added .lower() and .strip() to main.py, so the path argument would be filtered with those functions, then converted to a path object when put into the MockIMU class

Reply:
I had it as path object originally, but couldn't do .lower() and .strip() on path objects. I can just revert that though and keep the `type=Path`

Reply:
yeah revert them and keep it as type=Path. No need to post process either by doing .lower() and .strip().